homework/index.js

```diff
@@ -30,13 +30,122 @@
     return elem;
   }
 
+  function makeHeader(repos) {
+    const root = document.getElementById('root');
+    const header = createAndAppend('header', root, { class: 'header' });
+    header.style = 'display: flex;';
```

Comment:
Use your CSS file for styling.

Reply:
it was a code that I wrote for see a little better before I write CSS file. I deleted it.